main/src/mpu.cpp

```diff
@@ -0,0 +1,157 @@
+#include "mpu.h"
+#include "MPU6050.h"
+
+namespace mpu {
+
+MPU6050 mpudev;
+
+static SemaphoreHandle_t mutex = xSemaphoreCreateMutex();
+static unsigned int count = 0;
+static Data internal{};
+static float gyro_res = 0, accel_res = 0;
+static bool init_success = false;

```

Comment:
Why are those variables static? Are they actually supposed to be statically initialized and exist for the whole program lifetime or were you trying to make them translation unit local?

Reply:
It's an artifact from tests, removed.